### PR TITLE
Update docs for multi-bot LLM and name change.

### PR DESCRIPTION
**Pull request by crspeller**
#### Summary

Updates to the docs for multi-bot capabilities, name change from AI Copilot to Copilot, and removing Beta label for v10 release.

### CODE REVIEW

**Comment by cwarnermm** [comment]
Thank you, @crspeller! Is this docs PR associated with the v9.9 or v9.10 release?

**Comment by crspeller** [comment]
@cwarnermm The plugin is released now. It will be prepackaged with 9.10

**Comment by cwarnermm** [comment]
Thanks, @crspeller! I'm wondering whether multi-LLM support is releasing as part of v9.9 or v9.10.

**Comment by crspeller** [comment]
@cwarnermm So the multi-bot stuff will be prepackaged with 9.10, but I think we can update the docs whenever given that I think many users at this point are going to go to the repository to download the plugin. Not exactly sure what the best practice here would be.

**Comment by cwarnermm** [comment]
@crspeller - I agree that updating the terminology/naming needs to happen sooner than v9.10. I'm inclined to split this PR into 2: one for the language updates & one focusing on multi-LLM support queued for v9.10. I'll get the language updates into production early next week.

**Review by cwarnermm — APPROVED** [review]

File: source/collaborate/chat-with-copilot.rst [hunk 1]
```diff
@@ -8,11 +8,11 @@ Chat with Copilot (Beta)
   :alt: Select the AI Actions icon to access AI options.
   :class: theme-icon
 
-Overcome information overload and streamline communication and collaboration by chatting with Mattermost Copilot in your Mattermost instance.
+Overcome information overload and streamline communication and collaboration by chatting with the Mattermost Copilot in your Mattermost instance. Copilot is an LLM-powered assistant with a flexable backend that can be configured to meet your organization's needs. Copilot can help you summarize call recordings, threads, and unread channel messages, and provide insights on any topic you're curious about.
 
 .. note::
 
-  Copilot must be :doc:`enabled and configured </configure/enable-copilot>` by a Mattermost system admin in the System Console before you can start using it.
+  Copilot must be :doc:`enabled and configured </configure/enable-ai-copilot>` by a Mattermost system admin in the System Console before you can start using it.
```
Comment (azigler) [inline source/collaborate/chat-with-copilot.rst:15]: Is the URL changing?

File: source/collaborate/chat-with-copilot.rst [hunk 2]
```diff
@@ -90,12 +94,12 @@ Bring Copilot into any conversation
 .. include:: ../_static/badges/ent-only.rst
   :start-after: :nosearch:
 
-Invoke the power of AI by @mentioning the Copilot bot (@copilot) in any thread to bring AI's capabilities to your conversation, allowing for quick extraction of information or transformation of discussions into charts, resources, documentation, articles, and more. Copilot will find action items and open questions in new messages. With the power of Mattermost integrations and interoperability, the potential to enhance your workflow is limitless.
+Invoke the power of AI by @mentioning the Copilot bot with ``@copilot`` in any thread to bring AI's capabilities to your conversation, allowing for quick extraction of information or transformation of discussions into charts, resources, documentation, articles, and more. Copilot will find action items and open questions in new messages. With the power of Mattermost integrations and interoperability, the potential to enhance your workflow is limitless.
```
Comment (azigler) [inline source/collaborate/chat-with-copilot.rst:97]: Should we make it clear that `copilot` is the bot's username? Should we just use the default value of that field?

File: source/configure/enable-copilot.rst [hunk 3]
```diff
@@ -78,6 +78,43 @@ Configure a large language model (LLM) for your Copilot integration by going to
...
+
+Integrations
+~~~~~~~~~~~~
+
+Currently integrations are limited to DMs with the bots. The integrations will not operate from within a regular channel.
+
+Jira
+====
+
+Issues with public JIRA instances can be fetched. No configuration is required for this integration.
```
Comment (azigler) [inline source/configure/enable-copilot.rst:111]: Is this still the case? It can't access private Jira issues if authenticated?

**Comment by azigler** [comment]
@crspeller @cwarnermm Docs look great! Left a few comments and made some adjustments for readability and clarity.

File: source/configure/enable-copilot.rst (hunk 3 above)
Comment (crspeller) [reply source/configure/enable-copilot.rst]: Yes this is still the case.

File: source/collaborate/chat-with-copilot.rst (hunk 1 above)
Comment (crspeller) [reply source/collaborate/chat-with-copilot.rst]: Don't think so.

File: source/collaborate/chat-with-copilot.rst (hunk 2 above)
Comment (crspeller) [reply source/collaborate/chat-with-copilot.rst]: Updated

**Review by azigler — APPROVED** [review]
LGTM!

File: source/collaborate/chat-with-copilot.rst [hunk 4]
```diff
@@ -81,6 +83,8 @@ To summarize unread Mattermost channels:
 1. In a channel with unread messages, scroll to the **New Messages** cutoff, select **Ask AI**, and then select **Summarize new messages**.
 2. The channel summary is generated in the Copilot pane, and only you can view the summary.
 
+You can switch between multiple bots (if configured) by selecting them in the dropdown menu.
```
Comment (cwarnermm) [inline source/collaborate/chat-with-copilot.rst:86]:
```suggestion
When your system admin has :doc:`configured multiple bots </configure/enable-copilot>`, you can switch between them by selecting one from the drop-down menu.
```

File: source/configure/enable-copilot.rst [hunk 5]
```diff
@@ -78,6 +78,43 @@ Configure a large language model (LLM) for your Copilot integration by going to
   4. Enter your API key in the **API Key** field.
   5. Specify your model name in the **Default Model** field.
 
+Custom instructions
+====================
+
+Text input here is included in the prompt for every request. Use this to give your bots extra context or instructions. For example, you could list all of your organization's specific acronyms so the bot knows your vernacular and users can ask for definitions. Or you could give it specialized instructions such as adopting a specific personality or following a certain workflow. By customizing the instructions for each individual bot, you can create a more tailored AI experience for your specific needs.
+
+Enable Vision (Beta)
+=====================
```
Comment (cwarnermm) [inline source/configure/enable-copilot.rst:87]:
```suggestion
-------------------
```

File: source/configure/enable-copilot.rst [hunk 6]
```diff
@@ -78,6 +78,43 @@ Configure a large language model (LLM) for your Copilot integration by going to
   4. Enter your API key in the **API Key** field.
   5. Specify your model name in the **Default Model** field.
 
+Custom instructions
+====================
+
+Text input here is included in the prompt for every request. Use this to give your bots extra context or instructions. For example, you could list all of your organization's specific acronyms so the bot knows your vernacular and users can ask for definitions. Or you could give it specialized instructions such as adopting a specific personality or following a certain workflow. By customizing the instructions for each individual bot, you can create a more tailored AI experience for your specific needs.
+
+Enable Vision (Beta)
+-------------------
```
Comment (cwarnermm) [inline source/configure/enable-copilot.rst:87]:
```suggestion
---------------------
```

File: source/configure/enable-copilot.rst [hunk 7]
```diff
@@ -78,6 +78,43 @@ Configure a large language model (LLM) for your Copilot integration by going to
   4. Enter your API key in the **API Key** field.
   5. Specify your model name in the **Default Model** field.
 
+Custom instructions
+====================
+
+Text input here is included in the prompt for every request. Use this to give your bots extra context or instructions. For example, you could list all of your organization's specific acronyms so the bot knows your vernacular and users can ask for definitions. Or you could give it specialized instructions such as adopting a specific personality or following a certain workflow. By customizing the instructions for each individual bot, you can create a more tailored AI experience for your specific needs.
+
+Enable Vision (Beta)
```
Comment (cwarnermm) [inline source/configure/enable-copilot.rst:86]:
```suggestion
Enable vision (Beta)
```

File: source/configure/enable-copilot.rst [hunk 8]
```diff
@@ -78,6 +78,43 @@ Configure a large language model (LLM) for your Copilot integration by going to
...
 
+Custom instructions
+====================
+
+Text input here is included in the prompt for every request. Use this to give your bots extra context or instructions. For example, you could list all of your organization's specific acronyms so the bot knows your vernacular and users can ask for definitions. Or you could give it specialized instructions such as adopting a specific personality or following a certain workflow. By customizing the instructions for each individual bot, you can create a more tailored AI experience for your specific needs.
+
+Enable vision (Beta)
+---------------------
+
+Enabling vision allows images that are attached to posts to be sent to the upstream LLM for analysis. This requires that your upstream LLM support this features. Only available with OpenAI and OpenAI-compatable services.
```
Comment (cwarnermm) [inline source/configure/enable-copilot.rst:89]:
```suggestion
Enabling vision allows images that are attached to posts to be sent to the upstream LLM for analysis. This requires that your upstream LLM supports these features. Only available with OpenAI and OpenAI-compatable services.
```

File: source/configure/enable-copilot.rst [hunk 9]
```diff
@@ -78,6 +78,43 @@ Configure a large language model (LLM) for your Copilot integration by going to
...
+
+Enable vision (Beta)
+---------------------
+
+Enabling vision allows images that are attached to posts to be sent to the upstream LLM for analysis. This requires that your upstream LLM supports these features. Only available with OpenAI and OpenAI-compatable services.
+
+Copilot plugin metrics
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Metrics for the copilot plugin are exposed through the ``/plugins/mattermost-ai/metrics`` subpath under the existing Mattermost server metrics endpoint. This is controlled by the :ref:`Listen address for performance <configure/performance-monitoring-configuration-settings:listen address for performance>` configuration setting. It defaults to port ``8067``.
```
Comment (cwarnermm) [inline source/configure/enable-copilot.rst:94]:
```suggestion
Metrics for Copilot are exposed through the ``/plugins/mattermost-ai/metrics`` subpath under the existing Mattermost server metrics endpoint. This is controlled by the :ref:`Listen address for performance <configure/performance-monitoring-configuration-settings:listen address for performance>` configuration setting. It defaults to port ``8067``.
```

File: source/configure/enable-copilot.rst [hunk 10]
```diff
@@ -78,6 +78,43 @@ Configure a large language model (LLM) for your Copilot integration by going to
...
+
+Enable vision (Beta)
+---------------------
+
+Enabling vision allows images that are attached to posts to be sent to the upstream LLM for analysis. This requires that your upstream LLM supports these features. Only available with OpenAI and OpenAI-compatable services.
+
+Copilot plugin metrics
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Metrics for Copilot are exposed through the ``/plugins/mattermost-ai/metrics`` subpath under the existing Mattermost server metrics endpoint. This is controlled by the :ref:`Listen address for performance <configure/performance-monitoring-configuration-settings:listen address for performance>` configuration setting. It defaults to port ``8067``.
```
Comment (cwarnermm) [inline source/configure/enable-copilot.rst:94]:
```suggestion
Metrics for Copilot are exposed through the ``/plugins/mattermost-ai/metrics`` subpath under the existing Mattermost server metrics endpoint. This is controlled by the :ref:`Listen address for performance <configure/performance-monitoring-configuration-settings:listen address for performance>` configuration setting. It defaults to port ``8067``, and the following metrics are available:
```

File: source/configure/enable-copilot.rst [hunk 11]
```diff
@@ -78,6 +78,43 @@ Configure a large language model (LLM) for your Copilot integration by going to
...
+- ``copilot_system_plugin_info``: The plugin version and installation ID.
+- ``copilot_api_time_seconds``: How long to execute API.
+- ``copilot_http_requests_total``: The total number of API requests.
+- ``copilot_http_errors_total``: The total number of http API errors.
+- ``copilot_llm_requests_total``: The total number of requests to upstream LLMs.
+
+Integrations
+~~~~~~~~~~~~
+
+Currently integrations are limited to DMs with the bots. The integrations will not operate from within a regular channel.
```
Comment (cwarnermm) [inline source/configure/enable-copilot.rst:106]:
```suggestion
Currently integrations are limited to direct messages between users and the bots. The integrations won't operate from within public, private, or group message channels.
```

File: source/configure/enable-copilot.rst [hunk 12]
```diff
@@ -78,6 +78,43 @@ Configure a large language model (LLM) for your Copilot integration by going to
...
+
+Jira
+====
+
+Issues with public Jira instances can be fetched. No configuration is required for this integration.
+
+GitHub
+======
+
+If you have the GitHub plugin enabled, you can use the integration to fetch issues and PRs from your public and private Github repositories. The user is required to be logged in to GitHub though the plugin.
```
Comment (cwarnermm) [inline source/configure/enable-copilot.rst:116]:
```suggestion
If you have the Mattermost GitHub plugin enabled, you can use the integration to fetch issues and PRs from your public and private GitHub repositories. The user must be logged in to their GitHub account through the Mattermost GitHub plugin.
```

**Comment by cwarnermm** [comment]
Thank you @crspeller and @azigler for the team effort updating our Copilot documentation!!

**Comment by azigler** [comment]
I'm going to make some updates to the configuration page based on the [learnings from building the Academy course](https://hub.mattermost.com/private-core/pl/fzhai8pxnjnomm9ffbmce34una) (such as the Ollama URL and how to fetch the correct Azure information).

You'll see a follow-up commit from me later this week @cwarnermm @crspeller.

I also accounted for this in the [v10 release deliverables spreadsheet](https://docs.google.com/spreadsheets/d/136lh26ENaZkm0Xr1zwIZRgnCfVJJiAXDPuV8Y8ZK6Hg/edit?gid=1305144350#gid=1305144350).

**Comment by cwarnermm** [comment]
Very much appreciated, @azigler. Happy to support you wherever needed!

**Comment by azigler** [comment]
@cwarnermm The doc improvements based on the Academy videos are now done!

File: source/configure/enable-copilot.rst [hunk 13]
```diff
@@ -34,50 +34,97 @@ Mattermost configuration
...
+
+- ``copilot_system_plugin_start_timestamp_seconds``: The time the plugin started.
+- ``copilot_system_plugin_info``: The plugin version and installation ID.
+- ``copilot_api_time_seconds``: How long to execute API.
+- ``copilot_http_requests_total``: The total number of API requests.
+- ``copilot_http_errors_total``: The total number of http API errors.
+- ``copilot_llm_requests_total``: The total number of requests to upstream LLMs.
+
+Integrations
+~~~~~~~~~~~~
```
Comment (cwarnermm) [inline source/configure/enable-copilot.rst:114]:
```suggestion
---------------
```

File: source/configure/enable-copilot.rst [hunk 14]
```diff
@@ -34,50 +34,97 @@ Mattermost configuration
...
+
+Issues with public Jira instances can be fetched. No configuration is required for this integration.
+
+GitHub
+======
+
+If you have the Mattermost GitHub plugin enabled, you can use the integration to fetch issues and PRs from your public and private GitHub repositories. The user must be logged in to their GitHub account through the Mattermost GitHub plugin.
+
 Upgrade
 ~~~~~~~
```
Comment (cwarnermm) [inline source/configure/enable-copilot.rst:129]:
```suggestion
-----------
```

**Pull request merged**